src/config.test.ts

```diff
@@ -10,3 +10,18 @@ it("should get current version in case of missing schema URL", () => {
   const version = getVersionFromSchemaUrl(undefined);
   expect(version).toEqual(fabloVersion);
 });
+
+it("should check if version is supported", () => {
+  const currentMajorMinor = supportedVersionPrefix.slice(0, -1);
+  expect(isFabloVersionSupported(`${currentMajorMinor}.0`)).toBeTruthy();
+  expect(isFabloVersionSupported(`${currentMajorMinor}.999`)).toBeTruthy();
+  expect(isFabloVersionSupported("0.0.1")).toBeFalsy();
+});
+
+it("should check versions supporting raft", () => {
+  expect(versionsSupportingRaft("1.4.3")).toBeTruthy();
+  expect(versionsSupportingRaft("1.4.4")).toBeTruthy();
+  expect(versionsSupportingRaft("2.0.0")).toBeTruthy();
+  expect(versionsSupportingRaft("1.4.2")).toBeFalsy();
+  expect(versionsSupportingRaft("1.3.0")).toBeFalsy();
```

Comment:
Fablo actually dropped support for versions below `2.0`, and all versions support RAFT, so this test should be reworked, and also our checks in `./config` file